src/Microsoft.DotNet.Tools.Publish/PublishCommand.cs

```diff
@@ -193,6 +187,14 @@ private static void PublishFiles(IEnumerable<LibraryAsset> files, string outputP
                 }
 
                 File.Copy(file.ResolvedPath, Path.Combine(destinationDirectory, Path.GetFileName(file.ResolvedPath)), overwrite: true);
+                
+                // Copy pdbs
+                var pdbPath = Path.ChangeExtension(file.ResolvedPath, FileNameSuffixes.DotNet.ProgramDatabase);
```

Comment:
Wrap this in its own method instead of having that comment there. This and the next 4 lines into a method called CopyPds?